apps/server/lib/lexical/server/iex/helpers.ex

```diff
@@ -131,6 +131,21 @@ defmodule Lexical.Server.IEx.Helpers do
     |> Lexical.Server.Project.Supervisor.start()
   end
 
+  def time(fun) when is_function(fun, 0) do
+    start = System.monotonic_time()
+    result = fun.()
+    stop = System.monotonic_time()
+    duration = System.convert_time_unit(stop - start, :native, :millisecond)
+
+    IO.puts([
+      IO.ANSI.format([:cyan, :bright, "Time: "]),
+      to_string(duration),
+      "ms"
```

Comment:
this is also easy to do with `Lexical.Formats.time(microseconds)`
So this whole function can be reduced to:
```elixir
{elapsed_us, result} = :timer.tc(fun)
IO.puts(IO.ANSI.format([:cyan, :bright, "Time: ", Lexical.Format.time(elaped_us)])
result
```